### DISCUSSION TITLE
Support-file-missing-or-invalid

### DISCUSSION THREAD

**Discussion by LRejkowski** [cypress-io/cypress#22241] [Questions and Help]
Hi, 
from version 10x I have problem to run tests from Jenkis.
I'm getting:
**Couldn't find tsconfig.json. tsconfig-paths will be skipped
Your project does not contain a default supportFile. We expect a file matching cypress\support\e2e.{js,jsx,ts,tsx} to exist.**

even if files exist:
![image](https://user-images.githubusercontent.com/99488639/173012803-7fb00565-52d7-4ae6-8f9a-e1a78a1fde7d.png)

**Comment by fimonteiro** [comment]
Hey, you need to put in your cypress.config.js this config below:
```
const { defineConfig } = require("cypress");

module.exports = defineConfig({
  e2e: {
    supportFile: 'support/e2e.js',
    specPattern: 'e2e/**/*.cy.{js,jsx,ts,tsx}',
    setupNodeEvents(on, config) {
      // implement node event listeners here
    },
  },
});
```
Works well for me.

**Reply by LRejkowski** [reply]
Thank you!

**Reply by javierfuentesm** [reply]
hey my config is this one 
<img width="541" alt="image" src="https://user-images.githubusercontent.com/19244527/183240117-eb9edb17-484c-4f5c-97d5-8131ee392e00.png">
but im getting this : 
<img width="902" alt="image" src="https://user-images.githubusercontent.com/19244527/183240134-6baf8dc9-7f4e-4230-8a97-21e057b0bb0c.png">

**Reply by javierfuentesm** [reply]
@fimonteiro  any idea??  I'm not using jenkins but docker

**Reply by damian-sketch** [reply]
@javierfuentesm did you get a solution to this on docker?

**Reply by javierfuentesm** [reply]
@damian-sketch  Yes check this branch in my github https://github.com/javierfuentesm/CypressAvanzadoPlatzi/tree/22/docker
